### ISSUE TITLE
Overlapping instances for HasLogFunc SimpleApp

### ISSUE THREAD

**Issue by jeremyjh** [myfreeweb/magicbane#21]
In my init code, I wanted to use RIO's `runSimpleApp` for some code that needs a logging environment, before I've built my application context  - creating a sql connection pool wants a MonadLogger instance - and RIO doesn't export any function to write to console without a logger. But, I can't do that because there is an overlapping instance:

```
    • Overlapping instances for HasLogFunc SimpleApp
        arising from a use of ‘createPostgresqlPool’
      Matching instances:
        instance Has ModLogger α => HasLogFunc α
          -- Defined in ‘Magicbane.Logging’
        instance HasLogFunc SimpleApp -- Defined in ‘RIO.Prelude.Simple’
```

**Comment by akhra** [comment]
I think this is the same as #15, which is fixed on GitHub, but may never have made it to hackage?

**Comment by jeremyjh** [comment]
Thanks, master does indeed solve this. @myfreeweb any chance of a hackage release soon?

**Comment by valpackett** [comment]
released 0.4.0
